subprojects/platform-play/src/testFixtures/resources/org/gradle/play/integtest/fixtures/app/basicplayjavaapp/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'play'
+}
+
+model {
+    components {
+        play {
+            binaries.all {
+                tasks.withType(TwirlCompile) {
+                    javaProject = true
+                }
+            }
+        }
+    }
+}
+
+dependencies {
+    play "com.typesafe.play:play-java_2.11:2.3.9"
```

Comment:
I'm not completely happy with this dependency declaration, but I don't know how to access the default Scala and Play versions from `PlayPlatform` programmatically in this place. Any hints appreciated.